packages/cx/src/widgets/form/LookupField.js

```diff
@@ -821,9 +819,12 @@ class LookupComponent extends VDOM.Component {
 
    closeDropdown(e) {
       if (this.state.dropdownOpen) {
-         this.setState({
-            dropdownOpen: false
-         });
+         this.setState(
+            {
+               dropdownOpen: false
+            },
+            () => this.dom.input.focus()
```

Comment:
What is the reason behind this line?

It seems wrong because if you press Tab, the focus will move to the next element, the dropdown will close and then this will return focus to the lookup element.